### PR TITLE
Creating CircleCI workflow and jobs to build/deploy to S3

### PR DESCRIPTION
**Pull request by rleguen-broadsign**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+      - aws-cli/setup:
+          aws-region: AWS_REGION
+      - run:
+          name: Ensure infrastructure is up to date
+          command: |
+            aws cloudformation deploy \
+              --stack-name book-a-desk-ui \
+              --template-file ./infra/cf.yaml \
+              --capabilities
+  publish-to-s3:
```
Comment (TomasMorton) [inline .circleci/config.yml:40]: \n

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,7 +1,71 @@
...
+          at: ./
+      - run: ls -R dist/*
+      - aws-s3/sync:
+          arguments: --acl public-read
+          from: ./dist/
+          overwrite: false
+          to: s3://<< pipeline.parameters.s3-bucket >>/${CIRCLE_SHA1}
+
 workflows:
   node-tests:
```
Comment (TomasMorton) [inline .circleci/config.yml:54]: Should only run when not in release?

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,7 +1,71 @@
...
+    steps:
+      - checkout
+      - run:
+          name: NPM Build Project
+          command: |
+            npm run build
+      - persist_to_workspace:
+          root: ./dist
+          paths:
+            - ./*
```
Comment (TomasMorton) [inline .circleci/config.yml:24]: So this persists to ./dist, or to ./ when attached?

**Comment by rleguen-broadsign** [comment]
This PR wasn't updating with new commits, superseding it with a new PR: https://github.com/book-a-desk/book-a-desk-ui/pull/13

**Pull request closed without merging**